src/lib/fileutils.ts

```diff
@@ -595,7 +599,7 @@ const manifestGenerationPipelineYaml = () => {
           `download_spk`,
           `message="$(Build.SourceVersionMessage)"`,
           `if [[ $message == *"Merged PR"* ]]; then`,
-          `pr_id=$(echo $message | grep -oE '[0-9]+' | head -1 | sed -e 's/^0\+//')`,
+          `pr_id=$(echo $message | grep -oE '[0-9]+' | head -1 | sed -e 's/^0\\+//')`,
```

Comment:
@samiyaakhtar Was I correct in adding this additional `\`? 
without it, the `\` acts as a noop and the outputted string becomes `pr_id=$(echo $message | grep -oE '[0-9]+' | head -1 | sed -e 's/^0+//')`

Reply:
@evanlouie Looks good to me